swagger/index.js

```diff
@@ -131,13 +133,21 @@ module.exports = yeoman.Base.extend({
           name: model.name,
           plural: model.plural,
           base: model.base || 'PersistedModel',
-          facetName: 'server', // hard-coded for now
+          facetName: 'common', // hard-coded for now
```

Comment:
Why to change this?

If `server` is no longer a good default, then I think we should make this value configurable.